tests/sqlite/table.rs

```diff
@@ -328,16 +328,6 @@ fn drop_1() {
     );
 }
 
-#[test]
-fn truncate_1() {
-    assert_eq!(
-        Table::truncate()
-            .table(Font::Table)
-            .to_string(SqliteQueryBuilder),
-        r#"TRUNCATE TABLE "font""#
-    );
-}
-
```

Comment:
We can keep this test case and assert panic.

```rs
#[test]
#[should_panic(expected = "Sqlite doesn't support TRUNCATE statement")]
fn truncate_1() {
    assert_eq!(
        Table::truncate()
            .table(Font::Table)
            .to_string(SqliteQueryBuilder),
        r#"TRUNCATE TABLE "font""#
    );
}
```

Reply:
I have added back the test case